integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/ClusterTemplateAction.java

```diff
@@ -0,0 +1,78 @@
+package com.sequenceiq.it.cloudbreak.newway;
+
+import com.sequenceiq.it.IntegrationTestContext;
+import com.sequenceiq.it.cloudbreak.newway.log.Log;
+
+import java.io.IOException;
+
+class ClusterTemplateAction {
+
+    private ClusterTemplateAction() {
+    }
+
+    public static void post(IntegrationTestContext integrationTestContext, Entity entity) {
+        ClusterTemplateEntity clusterTemplateEntity = (ClusterTemplateEntity) entity;
+        CloudbreakClient client;
+        client = integrationTestContext.getContextParam(CloudbreakClient.CLOUDBREAK_CLIENT,
+                CloudbreakClient.class);
+        Long workspaceId = integrationTestContext.getContextParam(CloudbreakTest.WORKSPACE_ID, Long.class);
+        Log.log(" post "
```

Comment:
concatenation can be replaced by String.format like this:
`Log.log(String.format(" post %s cluster template. ", clusterTemplateEntity.getName()));`

not mandatory, just a hint :)